agenta-web/cypress/e2e/testset.cy.ts

```diff
@@ -22,6 +22,7 @@ describe("create a new testset", () => {
         const testsetName = randString(8)
         cy.get('[data-cy="testset-name-input"]').type(testsetName)
         cy.get('[data-cy="testset-save-button"]').click()
+        cy.wait(500)
```

Comment:
Why the `wait()` here?

Reply:
I tested withou it and ran successfully, removed :)